packages/rendermime/src/renderers.ts

```diff
@@ -496,9 +496,14 @@ export namespace renderSVG {
  *
  * @param content - The text content of a node.
  *
+ * @param replace - If URLs should be replaced.
+ *
  * @returns A list of text nodes and anchor elements.
  */
-function autolink(content: string): Array<HTMLAnchorElement | Text> {
+function autolink(
+  content: string,
+  replace: boolean
```

Comment:
Let's shunt this method to avoid unnecessary regex lookup.

```suggestion
  content: string
```